breathecode/certificate/tests/urls/tests_certificate_cohort_id_student_id.py

```diff
@@ -0,0 +1,224 @@
+"""
```

Comment:
change the file name from `tests_certificate_cohort_id_student_id.py` to `tests_cohort_id_student_id.py`

```python
    path('cohort/<int:cohort_id>/student/<int:student_id>',
         CertificateView.as_view(),
         name='certificate_single'),
```